single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DefaultSVGWriter.java

```diff
@@ -899,6 +898,71 @@ private void drawFeederInfo(String prefixId, String feederNodeId, List<Point> po
         root.appendChild(g);
     }
 
+    protected void insertBusInfo(String prefixId,
+                                 Point vlShift,
+                                 double shiftX,
```

Comment:
You can remove shiftX argument, drawBusInfo can get it directly from the parameters. It makes sense to keep it there if this shift was not the same for all the BusInfo, but I don't think it's useful so far.